.github/workflows/build-test-ubuntu20.yml

```diff
@@ -44,6 +44,28 @@ jobs:
           # not realy needed
           CMAKE_C_COMPILER: ${{ matrix.c-compiler }}
 
+      - name: Collect Timings
+        run: |
+          compilerName=${{fromJSON('["Clang_11", "GCC_10"]')[matrix.compiler == 'GCC 10']}}
```

Comment:
Well, the next (and I hope the last) iteration is to move the logic to the `./scripts/devops/collect_timings.sh` and to use this script on Linux and macOS